create/templates/.prettierrc.tpl

```diff
@@ -0,0 +1,7 @@
+{
+  "arrowParens": "always",
+  "bracketSpacing": false,
+  "jsxBracketSameLine": false,
+  "singleQuote": true,
+  "trailingComma": "es5"
+}
```

Comment:
This is a template file within `templates/` but doesn't do any actual templating (though maybe line 4 needs a React condition?). Does this approach make sense or would we rather do one of the following?

1.  Copy a non-templated version (.prettierrc) into each of the three create extension type directories
2. Change the source code to copy any non-templated files in the `/templates` directory to each extension directory
3. Leave it like this because even if there is no templating at the moment, it's a possibility moving forward

@vividviolet @t6d

Reply:
I'm for option number 3. Least work and most flexible. I don't mind that it is a `.tpl` file despite (currently) not using any template statements.